frontend/src/views/account-mgmt/FacilityView.vue

```diff
@@ -0,0 +1,271 @@
+        <v-card class="pa-6 w-100" variant="outlined">
+          <v-skeleton-loader :loading="loading" type="table-tbody">
+            <div class="w-100"> <!-- NOTE: div was required due to dynamic v-row within v-skeleton-loader, otherwise intended row formatting breaks -->
+              <v-row>
+                <v-col>
+                  <AppLabel>Current expense authority:</AppLabel>
+                </v-col>
+                <v-col>
+                  <v-row no-gutters justify="end">
+                    <v-icon icon="fa:fa-regular fa-plus" class="mr-4"></v-icon>
+                    <v-icon v-if="isEditableExpenseAuthority" icon="fa:fa-regular fa-edit" class="mr-4"></v-icon>
+                  </v-row>
+                </v-col>
+              </v-row>
+              <v-row v-for="(item, index) in this.expenseAuthorities" :key="index">
+                <v-col cols="auto">
+                  <AppLabel>Name: </AppLabel>
+                </v-col>
+                <v-col cols="2">
+                  {{ item.firstName }} {{ item.lastName }}
+                </v-col>
+                <v-col cols="auto">
+                  <AppLabel>Role:</AppLabel>
+                </v-col>
+                <v-col cols="2">
+                  {{ getRoleNameById(item.role) }}
+                </v-col>
+              </v-row>
+            </div>
+          </v-skeleton-loader>
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col class="ml-6 pr-9 pt-0 pb-0">
+        <h4>Primary Contact</h4>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12" class="ml-6 pr-9 pt-0">
+        <v-card class="pa-6 mb-3" variant="outlined">
+          <v-skeleton-loader :loading="loading" type="table-tbody">
+            <div class="w-100"> <!-- NOTE: div was required due to dynamic v-row within v-skeleton-loader, otherwise intended row formatting breaks -->
+              <v-row>
+                <v-col>
+                  <AppLabel>Current primary contact info:</AppLabel>
+                </v-col>
+                <v-col>
+                  <v-row no-gutters justify="end">
+                    <v-icon v-if="isEditablePrimaryContact" icon="fa:fa-regular fa-edit" class="mr-4"></v-icon>
+                  </v-row>
+                </v-col>
+              </v-row>
+              <v-row v-for="(item, index) in this.primaryContact" :key="index">
+                <v-col cols="auto">
+                  <AppLabel>Name: </AppLabel>
+                </v-col>
+                <v-col cols="2">
+                  {{ item.firstName }} {{ item.lastName }}
+                </v-col>
+                <v-col cols="auto">
+                  <AppLabel>Role:</AppLabel>
+                </v-col>
+                <v-col cols="auto">
+                  {{ getRoleNameById(item.role) }}
+                </v-col>
+              </v-row>
+            </div>
+          </v-skeleton-loader>
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="10">
```

Comment:
The button alignment needs a little work at lower resolutions. This seems to work well:

    <v-row>
      <v-col cols="12" md="6">
        <v-row justify="center" justify-md="start" class="pb-2">
          <AppButton id="back-to-account-mgmt" :primary="false" size="medium" width="400px" :to="{ name: 'account-mgmt' }" :loading="loading">&larr; Back to Account Management</AppButton>
        </v-row>
      </v-col>
      <v-col cols="12" md="6">
        <v-row justify="center" justify-md="end" class="pb-2">
          <AppButton id="cancel" :primary="true" size="large" :to="{ name: 'organization-facility' }" :loading="loading" class="mr-6">Cancel</AppButton>
          <AppButton id="save" :disabled="true" :primary="true" size="large" :to="{ name: 'account-mgmt' }" :loading="loading">Save</AppButton>
        </v-row>
      </v-col>
    </v-row>

Reply:
updated.